Library/src/CSP/Systems/EventTicketing/EventTicketing.cpp

```diff
@@ -81,6 +102,73 @@ void SpaceEventDtoToTicketedEvent(const chs::SpaceEventDto& Dto, csp::systems::T
 	}
 }
 
+void SpaceTicketDtoToEventTicket(const chs::SpaceTicketDto& Dto, csp::systems::EventTicket& Ticket)
+{
+	Ticket.Id	   = Dto.GetId();
```

Comment:
If you assigned `Ticket` a default-constructed `EventTIcket` object at the top here, you could avoid having to assign default values for all cases where a dto property was missing?

Reply:
This is following a pattern that we use in many similar places, will take a look and see if it makes sense to do this.

Reply:
Understood, don't be swayed by the established pattern if this approach appears to be more future-proof!

Reply:
Looking around at a few other `*DtoTo*` functions that we have, we actually often do not even log missing values. However, I think there is value in logging and setting defaults for values that we expect. I think it makes sense for those defaults to be set here as I don't think this code should assume defaults that are set in the constructor are correct in this situation.

Reply:
Ok